core/domain/state_domain.py

```diff
@@ -564,6 +620,14 @@ def get_all_html_content_strings(self):
         if self.solution:
             solution_html = self.solution.explanation.html
             html_list = html_list + [solution_html]
+            if self.solution.correct_answer:
+                interaction = (
+                    interaction_registry.Registry.get_interaction_by_id(
+                        self.id).to_dict())
+                if (interaction['answer_type'] ==
+                        feconf.ANSWER_TYPE_LIST_OF_SETS_OF_HTML):
```

Comment:
Should we add this constant in the interaction which uses this asnwer_tye?

Reply:
changed this